views/stats/user.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3.5
+# encoding: utf-8
+from sanic.response import json
+
+from views.utils import user_required
+from views.utils import HTTPModelClassView
+from models import Users
+
+
+class UserStatsView(HTTPModelClassView):
+    _cls = Users
+    _urls = '/api/users_stats'
+
+    @user_required('admin')
+    async def get(self, _, current_user):
```

Comment:
mam wrażenie, że trochę dużo tu redundancji; postaram się później pomyśleć jak to trochę uprościć

Reply:
tak jest jeszcze redundancja zmniejszenie jej bedzie w part/epizodzie II